src/hooks/useMetamask.ts

```diff
@@ -50,7 +50,7 @@ async function connect(): Promise<IMetamaskStoreState> {
   return newState;
 }
 
-async function requestAccount() {
+export async function requestAccount() {
```

Comment:
this is an internal function. if you want to get the address, you should use the `connect` method from `useMetamask`